### PR TITLE
Sync develop changes since last sync to hdf5_1_14

### PR DESCRIPTION
**Pull request by lrknox**

### CODE REVIEW

File: .github/workflows/abi-report.yml
Comment (byrnHDF) [inline .github/workflows/abi-report.yml:1]: This will need fixed for 1.14

File: .github/workflows/cmake-bintest.yml
Comment (byrnHDF) [inline .github/workflows/cmake-bintest.yml:1]: This will need fixed for 1.14

File: .github/workflows/main-auto-par-spc.yml
Comment (byrnHDF) [inline .github/workflows/main-auto-par-spc.yml:1]: This will need fixed for 1.14

File: .github/workflows/main-cmake-par.yml
Comment (byrnHDF) [inline .github/workflows/main-cmake-par.yml:1]: This will need fixed for 1.14

File: release_docs/INSTALL_Auto.txt
Comment (byrnHDF) [inline release_docs/INSTALL_Auto.txt:1]: new files

File: release_docs/INSTALL_CMake.txt
Comment (byrnHDF) [inline release_docs/INSTALL_CMake.txt:1]: new files

**Review by byrnHDF — APPROVED** [review]
Some files will need fixed afterwards.
CMake CI approved.

**Review by jhendersonHDF — APPROVED** [review]

**Pull request merged**